CSS/ImdbCSS.css

```diff
@@ -0,0 +1,28 @@
+body {

```

Comment:
@oregilron File name should not be camelCased also the file extension is css so you don't need to mention it also in the file name.